client/app/utils/game-channel.js

```diff
@@ -0,0 +1,32 @@
+import { Socket } from 'vilive/utils/phoenix';
+import ENV from 'vilive/config/environment';
+
+let socket = null;
+let channel = null;
+
+export default {
+  init (token) {
+    socket = new Socket(ENV.yggdrasil.socket + '/socket',
+                        {params: {token: token}});
+    socket.connect();
+
+    // Now that you are connected, you can join channels with a topic:
+    channel = socket.channel("game:lobby", {"token": token});
+    channel.join()
+      .receive("ok", resp => { console.log("Joined successfully", resp); })
+      .receive("error", resp => { console.log("Unable to join", resp); });
```

Comment:
This code looks out of date. It isn't creating the player's channel and joining the player's topic. If you use the client you'll notice that it doesn't display a welcome message.